tech-details.md

```diff
@@ -74,14 +74,80 @@ Open `html/portal.html` in your browser to preview changes. The contents of `htm
 
 **Web portal budget:** Right now the portal costs about 18KB. We'll allow up to **100KB**. All portal assets must be locally available, since the portal needs to work when the device doesn't have WiFi. If you want to try adding a more complex UI framework than hand-rolled JS and HTML, you're welcome to try as long as its build system is quick and the final static output is under 100KB.
 
-### Cards
+### Card system
+
+The DeskHog uses a dynamic card system that allows users to configure which cards appear on their device through a web UI. Cards are managed by the `CardController` and can be easily extended by developers.
```

Comment:
Do we want to clarify here that all cards exist on the system anyway, and that the card manager is just controlling the UI?